source/Generic/PlayState/PlayState.cs

```diff
@@ -120,6 +119,14 @@ public override void OnApplicationStarted(OnApplicationStartedEventArgs args)
                     .Show();
                 ProcessStarter.StartUrl(@"https://github.com/darklinkpower/PlayniteExtensionsCollection/wiki/PlayState#window-notification-style-configuration");
             }
+
+            var suspendPlaytimeOnlyFeature = PlayniteApi.Database.Features.FirstOrDefault(a => a.Name.Equals("[PlayState] Suspend Playtime only", StringComparison.OrdinalIgnoreCase));
+            if (suspendPlaytimeOnlyFeature != null)
+            {
+                suspendPlaytimeOnlyFeature.Name = "[PlayState] Override suspend playtime only setting";
```

Comment:
I think the string is too long, ideally it should be short so at least most part of it can be read on the Filter Panel 
![image](https://user-images.githubusercontent.com/1389286/160934742-c84b1ba7-d914-4f0d-8f34-d015b2dc3d41.png)